integration-tests/api-gateway/endpoints.ts

```diff
@@ -29,6 +29,16 @@ export const createPayload = async () => {
   preOutput = await stackOutputs(process.env.STACK_NAME);
   privateAPI = `${preOutput.PrivateApiGatewayId}`;
   const correctClaimSet = await getClaimSet();
+
+  if (useMultipleNames = true){
```

Comment:
Just thinking about future uses - maybe we want to be able to optionally pass in an entire shared_claims set? This allows us to easily test different sets of user data without having to add a whole load of flags

Reply:
I've removed if statement and updated the createPayload to take in the updated `name`

Reply:
Nice thank you!